README.md

```diff
@@ -40,42 +40,76 @@ NuGet: Install-Package RestClient.NET
 
+
 ### [Get](https://github.com/MelbourneDeveloper/RestClient.Net/blob/13c95c615400d39523c02e803b46a564ff4c91db/RestClient.Net.UnitTests/UnitTests.cs#L81)
 
 With [Newtonsoft serialization](https://github.com/MelbourneDeveloper/RestClient.Net/wiki/Serialization-and-Deserialization-With-ISerializationAdapter#newtonsoft)
 
+<!-- snippet: GetNewtonsoft -->
+<a id='snippet-getnewtonsoft'></a>
 ```cs
 var client = new Client(new NewtonsoftSerializationAdapter(), new Uri("https://restcountries.eu/rest/v2/"));
 var response = await client.GetAsync<List<RestCountry>>();
 ```
+<sup><a href='/src/RestClient.Net.UnitTests/Snippets.cs#L28-L33' title='Snippet source file'>snippet source</a> | <a href='#snippet-getnewtonsoft' title='Start of snippet'>anchor</a></sup>
+<!-- endSnippet -->
 
 With [default serialization on .NET Core](https://github.com/MelbourneDeveloper/RestClient.Net/wiki/Serialization-and-Deserialization-With-ISerializationAdapter#default-json-serialization-adapter)
+
+<!-- snippet: GetDefault -->
+<a id='snippet-getdefault'></a>
 ```cs
-var client = new Client(new Uri("https://restcountries.eu/rest/v2/"));
+var client = new Client(baseUri: new Uri("https://restcountries.eu/rest/v2/"));
 var response = await client.GetAsync<List<RestCountry>>();
 ```
+<sup><a href='/src/RestClient.Net.UnitTests/Snippets.cs#L41-L46' title='Snippet source file'>snippet source</a> | <a href='#snippet-getdefault' title='Start of snippet'>anchor</a></sup>
+<!-- endSnippet -->
+
 
 ### Post / Put / Patch
 
-[**Protocol Buffers**](https://github.com/MelbourneDeveloper/RestClient.Net/blob/80d19ebc599027e2c68acb06a4e1f853683c3517/RestClient.Net.Samples/RestClient.Net.CoreSample/Program.cs#L25) (Binary)
+
+#### Protocol Buffers (Binary)
+
+<!-- snippet: PostBinary -->
+<a id='snippet-postbinary'></a>
 ```cs
 var person = new Person { FirstName = "Bob", Surname = "Smith" };
 var client = new Client(new ProtobufSerializationAdapter(), new Uri("http://localhost:42908/person"));
 person = await client.PostAsync<Person, Person>(person);
 ```
+<sup><a href='/src/RestClient.Net.UnitTests/Snippets.cs#L53-L59' title='Snippet source file'>snippet source</a> | <a href='#snippet-postbinary' title='Start of snippet'>anchor</a></sup>
+<!-- endSnippet -->
+
 
-[**JSON**](https://github.com/MelbourneDeveloper/RestClient.Net/blob/236a454232455aa3dc0cea230e991329288c153d/RestClient.Net.Samples/RestClient.NET.Samples/MainPage.xaml.cs#L233)
+#### JSON
+
+<!-- snippet: PostNewtonsoft -->
+<a id='snippet-postnewtonsoft'></a>
 ```cs
 var client = new Client(new NewtonsoftSerializationAdapter(), new Uri("https://jsonplaceholder.typicode.com"));
-client.SetJsonContentTypeHeader();
-UserPost userPost = await client.PostAsync<UserPost, UserPost>(new UserPost { title = "Title" }, "/posts");
+//TODO: client.SetJsonContentTypeHeader();
```

Comment:
@MelbourneDeveloper SetJsonContentTypeHeader doesnt seem to exist?

Reply:
Lol. This is what I mean...

The documentation you are looking at is for V4. It's the NuGet package that is currently live. main is in sync with the NuGet package.

This has all changed in V5.

Reply:
in this scenario, in the rare case where someone asks, i regularly deploy betas so the docs match the beta. and if people want the old docs i send the to a tag. for me a working on a new major is a small time window, so the slight mismatch isnt a problem